pom.xml

```diff
@@ -181,15 +184,15 @@
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-javadoc-plugin</artifactId>
-					<version>2.10.3</version>
+					<version>3.0.0</version>
 					<executions>
 						<execution>
 							<id>attach-javadocs</id>
 							<goals>
 								<goal>jar</goal>
 							</goals>
 							<configuration>
-								<additionalparam>-Xdoclint:none</additionalparam>
+								<additionalJOption>-Xdoclint:none</additionalJOption>
```

Comment:
why let people of the hook by default? Shouldn't they at least see these warnings and error messages before disabling them?